.github/workflows/tests.yml

```diff
@@ -75,6 +76,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install '.[test]'
+          pip install --upgrade git+https://github.com/PennyLaneAI/pennylane.git#egg=pennylane
```

Comment:
Should we keep the ordering identical between tests and integration-tests :thinking: